src/main/groovy/com/blackducksoftware/integration/hub/detect/hub/HubServiceWrapper.groovy

```diff
@@ -114,6 +103,23 @@ class HubServiceWrapper {
         return false;
     }
 
+    public void assertHubConnection() {
```

Comment:
We should kill this method and just use `assertHubConnection(new SilentLogger()`